pep-9999.rst

```diff
@@ -47,13 +47,46 @@ required to interpret all standard constructs, and additionally a type checker
 can support extra constructs not described in this PEP. All deviations from
 this PEP should be documented.
 
+Syntax
+======
+
+Type stubs are syntactically valid Python files with a ``.pyi`` suffix.
+They should be valid according to the latest Python version, but also to
+the previous Python version up until six months after
+the release of the current version.
+Type stubs authors are encouraged to use the latest available syntax
+features in stubs, even if the type information in the stub is targeted
+to an older Python version.
+
+Type checkers are expected to support the syntax of the latest released Python
+version.
+
+For example, Python 3.6, introducing PEP 526 [#pep526]_variable annotations,
+was released on December 23rd, 2016. Type checkers should support
+variable annotations by that date, but type stubs should only
+started using them by June 23rd, 2017. Since that date the use
+of variable annotations is recommended for stubs, even if the implementation
+supports Python versions older that 3.6.
+
+Distribution
+============
+
+Type stubs can be distributed separately from the implementation;
+see PEP 561 [#pep561]_ for more information. The typeshed_ project
+includes stubs for Python's standard library and several third-party
+packages. These are usually distributed with type checkers and do not
+require separate installation.
+
 References
 ==========
 
 .. [#pep484] PEP 484 -- Type Hints, van Rossum et al. (https://www.python.org/dev/peps/pep-0484)
 .. [#pep526] PEP 526 -- Syntax for Variable Annotations, Gonzalez et al. (https://www.python.org/dev/peps/pep-0526)
+.. [#pep561] PEP 561 -- Distributing and Packaging Type Information, Smit (https://www.python.org/dev/peps/pep-0561)
```

Comment:
Unless you are intentionally truncating this, should be Smith :)

Reply:
Whoops, sorry, fixing.